Gemfile.lock

```diff
@@ -371,3 +371,9 @@ DEPENDENCIES
   tzinfo-data
   web-console
   webdrivers
+
+RUBY VERSION
+   ruby 3.1.0p0
+
+BUNDLED WITH
+   2.3.9
```

Comment:
Unsure what is causing this.
I've tried deleting the .lock, and copying the Gemfile from v3. 
It still does this.